src/Controls/src/Core/HandlerImpl/MenuItem.Impl.cs

```diff
@@ -1,12 +1,13 @@
 #nullable disable
+using System.Collections.Generic;
 using Microsoft.Maui.Graphics;
 
 namespace Microsoft.Maui.Controls
 {
 	/// <include file="../../../docs/Microsoft.Maui.Controls/MenuItem.xml" path="Type[@FullName='Microsoft.Maui.Controls.MenuItem']/Docs/*" />
 	public partial class MenuItem : IMenuElement
 	{
-		IAccelerator IMenuElement.Accelerator => GetAccelerator(this);
+		IReadOnlyList<IAccelerator> IMenuElement.Accelerators => new[] { GetAccelerator(this) };
```

Comment:
We should not create an array with a potentially null item in it.

```suggestion
		IReadOnlyList<IAccelerator> IMenuElement.Accelerators =>
			GetAccelerator(this) is Accelerator acc
				? new[] { acc }
				: Array.Empty<Accelerator>();
```

Reply:
Actually, depending on the vote of my comment below:

> Do we want to have a null collection, or an empty collection?

The else part should switch to null if we want nulls.

Reply:
I vote for empty, but either works for me! 

@jsuarezruiz @PureWeen thoughts?

Reply:
I'd vote for allowing null on the interface to handle future worlds where we might want this to be null in core but we can return an empty array from controls.